DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer/README.md

```diff
@@ -300,10 +300,10 @@ You may also build the SOF using the pre-generated Intel® Qupartus® Prime proj
 ### Additional Documentation
 - [Intel® Arria® 10 SoC Golden System Reference Design](https://rocketboards.org/foswiki/Documentation/Arria10SoCGSRD) describes a reference design you can use with your Intel® Arria® 10 SX SoC Developer kit.
 - [Intel® Arria® 10 SX SoC Development Kit](https://www.intel.com/content/www/us/en/products/details/fpga/development-kits/arria/10-sx.html) describes the Intel® Arria® 10 SX SoC Development kit in greater detail.
-- [Intel® FPGA Software Installation and Licensing](https://www.intel.com/content/www/us/en/docs/programmable/683472/current/) describes how to license Intel® Quartus® Prime Pro Edition (or Standard Edition when targeting Cyclone® V) software.
-- [Intel® Quartus® Prime Pro Edition (or Standard Edition when targeting Cyclone® V) User Guide: Getting Started](https://www.intel.com/content/www/us/en/docs/programmable/683463/current/) introduces you to the Intel® Quartus® Prime Pro Edition (or Standard Edition when targeting Cyclone® V) software.
```

Comment:
looks like yohann did a find-replace job here haha... those links def do not point to standard edition :)